packages/bootstrap/scss/grid/_theme.scss

```diff
@@ -133,4 +133,31 @@
     .k-column-list-wrapper {
         border-color: inherit;
     }
+
+    .k-column-menu {
```

Comment:
`.k-column-menu` is the popup. The styles below should be applied here.

Considering the original styles of the menu, perhaps we should rework it for the default theme:

```scss
.k-menu {} // styles for menu
.k-context-menu {} // styles for context menu
.k-column-menu {} // styles for column menu
```

OR, we should switch to using context menu there, because that's what we want. @PreslavKozovski could you assist for that?